task_16/test/specs/main.js

```diff
@@ -0,0 +1,93 @@
+describe('Hard core', () => {
+    it('Open Google Cloud Homepage', async () => {
+        await browser.url('/')
+
+        //Find and click on search button
+        const searchBtn = await $('.devsite-searchbox > input')
+        await searchBtn.waitForExist({timeout: 5000})
+        await searchBtn.click()
+
+        //Type into search bar
+        await searchBtn.addValue('Google Cloud Platform Pricing Calculator')
+        await browser.keys("\uE007")
+
+        //Find result
+        const searchResult = await $('a.gs-title > b')
+        await searchResult.waitForExist({timeout: 5000})
+        await searchResult.click()
+        
+        //Activate Compute Engine
+        browser.waitUntil(() => browser.execute(() => document.readyState === 'complete'),
+            {
+              timeout: 60 * 1000, // 60 seconds
+              timeoutMsg: 'Message on failure'
+            }
+        )
+        const computeEngine = await $('div*=Compute')
+        await computeEngine.waitForExist({timeout: 5000})
+        await computeEngine.click()
+
+        //Number of instances 4
+        const closeBurgerBtn = await $('/html/body/section/devsite-header/div/div[1]/div/div/div[2]/div[1]/cloudx-tabs-nav/nav/tab[2]/div/button')
+        await closeBurgerBtn.waitForExist({timeout: 30000})
+        await closeBurgerBtn.click()
+        const numOfInstances = await $('//*[@id="input_77"]')
+        await numOfInstances.waitForExist({timeout: 30000})
+        await numOfInstances.addValue(4)
+
+        //Select machine type
+        const machineTypeInput = await $('//*[@id="select_value_label_73"]/span[2]')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887245)
```suggestion
        const machineTypeInput = await $("//*[@id=\"select_value_label_73\"]/span[2]")
```